portfolio/src/main/webapp/index.html

```diff
@@ -1,33 +1,101 @@
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <div id="content">
+  <h1>Meagan Konst's Portfolio</h1>
+  <img src="/images/Selfie.jpg" alt="Meagan Selfie" width="300" height="400" class="center">
+  <div id="smallContent">
       <h2>Intro</h2>
-      <p>I am an Electrical and Computer Engineering double major at Washington University in St. Louis who loves to teach and play bass guitar.</p>
-      <h2>Random Fact</h2>
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Random Fact</button>
-      <div id="random-fact-container"></div>
-      <h2>My Background</h2>
-      <p>I started my education at a public school called Brooklands Elementary School that was within walking distance of my house. My favorite thing about going to Brooklands was that all of 
-          my school friends lived in my neighborhood or close by. When my dad suggested that I apply to The Roeper School, I was hesitant to leave mainly because I would miss my 
-          friends that I had known since kindergarten. This was why I did not end up transferring for 4th grade and decided to hold off until 5th. Ultimately this was one of the best decisions 
-          that I've made. Roeper allowed me to be challenged and engaged in my learning and be a part of a tight-knit community that supported me through my senior year and beyond. My time at 
-          Roeper played a large role in forming who I am today. Their philosophy praises teaching the whole child, not simply grooming applicants for college admissions or standardized tests. 
-          The small size and general no-cut policy allowed me to try different things and be involved in as many extra-curriculars as I'd like. I cherish each opportunity I've had and attempt to 
-          pay it forward by helping where I can, whether it be tutoring a student in my free time, or becoming a Teaching Assistant for a course, or simply being a good friend. This goal drives 
-          me, I truly enjoy being helpful.
-          </p>
-      <h2>My Pets</h2>
-      <p>I love my pets and I hope you enjoy their portraits below:</p>
+    <p>I am an Electrical and Computer Engineering double major at Washington University in St. Louis who loves
+         to teach and play bass guitar.
+    </p>
+    <h2>Random Fact</h2>
+    <p>Click here to learn a random fact about me:</p>
+    <button onclick="addRandomFact()">Random Fact</button>
+    <div id="random-fact-container"></div>
+      <h2>My Favorites</h2>
+      <p>Click here to learn one of my favorite things:</p>
+      <button onclick="addRandomFavorite()">Favorite Thing</button>
+      <div id="random-favorite-container"></div>
+    <h2>My Background</h2>
+    <p>I started my education at a public school called Brooklands Elementary School that was within walking
+           distance of my house. My favorite thing about going to Brooklands was that all of
```

Comment:
Please re-indent this section to match the `>` in `<p>`. Example here: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Generally, everything should be indented two spaces from the enclosing block.